controllers/api/index.js

```diff
@@ -1,5 +1,6 @@
-const router = require('express').Router();
```

Comment:
This line is required as the express server was already set up in server.js to require and use the items in the controllers folder.